packages/core/src/Cardano/types/Genesis.ts

```diff
@@ -1,22 +1,19 @@
-import { Epoch, Int64, NetworkMagic, UInt64 } from '@cardano-ogmios/schema';
 import { Lovelace } from '.';
-import { NetworkId } from '../NetworkId';
-
-export type Ratio = number;
+import { NetworkId, NetworkMagic } from '../NetworkId';
 
 /**
- * A compact (without genesis UTxO) representation of the genesis configuration.
+ * A compact (without genesis UTxO) representation of the genesis configunumbern.
```

Comment:
:laughing: - Find replace fail

Reply:
fixed (I am squashing as I make this final changes)